lib/fizzy/execute.cpp

```diff
@@ -485,6 +485,26 @@ inline uint64_t popcnt64(uint64_t value) noexcept
     return static_cast<uint64_t>(__builtin_popcountll(value));
 }
 
+template <typename T>
+inline constexpr T fmin(T a, T b) noexcept
+{
+    if (std::isnan(a))
+        return a;
+    if (std::isnan(b))
```

Comment:
Are there any tests deciding this order? e.g. what happens if both are nan?

Reply:
Yes. The standard NaN propagation rules apply.